packages/components/pie-modal/playwright-lit-visual.config.ts

```diff
@@ -1,4 +1,3 @@
-import { defineConfig } from '@sand4rt/experimental-ct-web';
-import { getPlaywrightVisualConfig } from '@justeattakeaway/pie-components-config';
+import { defineVisualConfig } from '@justeattakeaway/pie-components-config';
```

Comment:
Thanks for the PR! Should this be `definePlaywrightVisualConfig`?

Reply:
fixed